### ISSUE TITLE
Take the description of using the bouncer from Colloquy out of the FAQ <_<

### ISSUE THREAD

**Issue by akempgen** [colloquy/colloquy#2472]
*Issue migrated from trac ticket [#2319](http://colloquy.info/project/ticket/2319)*

**type:** Task | **component:** Colloquy (Mac) | **severity:** Major | **priority:** High | **version:** 2.3 (Mac) | **status:** closed | **resolution:** Works for Me
___
#### 2010-10-14 23:21:58: anonymous created the issue

cf. ticket 2193. Why is that even on the Wiki, it's completely wrong, just wasting people's time...
___
#### 2010-10-15 01:53:13: @zadr changed status from *new* to *closed*

___
#### 2010-10-15 01:53:13: @zadr set resolution to *worksforme*

___
#### 2010-10-15 01:53:13: @zadr commented

It is docoumented because the question comes up every so often, and we prefer to not lie to people. Especially when they can look at the source and catch us.

Basically, it technically does work. Its just finicky. Its explicitly mentioned as being unsupported.
